Firestore/Example/Firestore.xcodeproj/project.pbxproj

```diff
@@ -5986,6 +5986,7 @@
 				ONLY_ACTIVE_ARCH = YES;
 				OTHER_CFLAGS = "";
 				SDKROOT = iphoneos;
+				SWIFT_VERSION = 5.0;
```

Comment:
This one is 5.0 instead of 5.9 is because it is generated from UI Config, and Xcode only allow to choose between 5.0 and 6.0